### PR TITLE
Revisions2021

### PR DESCRIPTION
**Pull request by rosherbal**
Added suggestions from reviewers.
This includes:
- Included a downloadable report for each of the tools with the parameters, used code and the figures of the analysis.
- The user can now use any type of ids for the genes provided. Additionally, when uploading a different expression file, they can also select over which statistic the analysis would be performed.
- We have included an explicit list of dependencies. This can be consulted in https://github.com/saezlab/ShinyFUNKI/FUNKI/list_of_dependencies.txt
- The examples are now also downloadable.

### CODE REVIEW

File: FUNKI/data/reports/dorothea_report.Rmd [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- organism: `r params$organism`
+- min size: `r params$minsize`
+- method: `r params$method`
+- confidence level: `r paste(params$selected_conf_level, collapse = ",")`
+
+The code:
+
+```{r, eval=FALSE}
+dorothea::run_viper(dorothea_matrix, 
+                    regulons,
```
Comment (christianholland) [inline FUNKI/data/reports/dorothea_report.Rmd:45]: How is the variable `params$selected_conf_level` reflected in the choice of the regulons? I would expect a filter step here

File: FUNKI/data/reports/kinact_report.Rmd [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  selected_kinase: NA
+  selected_sample: NA
+  numberKinases: NA
+  sample_plot: NA
+  tf_plot: NA
+  network_plot: NA
+  heatmap_plot: NA
+---
+
+[KinAct](http://saezlab.github.io/kinact/) is a resource linking kinases to phosphorylation sites ( [Wirbel et al., 2018](https://www.biorxiv.org/content/10.1101/066019v1) ).
```
Comment (christianholland) [inline FUNKI/data/reports/kinact_report.Rmd]: Are you sure that the biorxiv version is the most recent publication about kinact?

File: FUNKI/data/reports/dorothea_report.Rmd (hunk 1 above)
Comment (rosherbal) [reply FUNKI/data/reports/dorothea_report.Rmd:45]: fixed in the last commit

File: FUNKI/data/reports/kinact_report.Rmd (hunk 2 above)
Comment (rosherbal) [reply FUNKI/data/reports/kinact_report.Rmd]: fixed with the last commits

**Pull request merged**